unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py

```diff
@@ -1402,5 +1400,1381 @@ def test_optional_feild_app_meta_data_schema(self):
         self.assertEqual(actual, expected)
         logger.debug("-->Test was Successfully verified")
 
+    def test_network_feild_compatilibity(self):
```

Comment:
typo
`feild_compatilibility` should be `field_compatibility`

Reply:
thank you, I will correct and update